confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, url, max_schemas_per_subject=1000, ca_location=None, cert_loc
             """Construct a Schema Registry client"""
 
         # Ensure URL valid scheme is included; http[s]
-        url = conf.get('url', '')
+        url = conf.pop('url')
```

Comment:
Keep the default, '', to avoid throwing KeyError.  The check validation below will throw a more appropriate ValueError when it finds the url is not prepended with 'http'.

Reply:
Fixed)